attest/ake/src/state.rs

```diff
@@ -110,3 +110,10 @@ where
 }
 
 impl<Cipher> State for Ready<Cipher> where Cipher: NoiseCipher {}
+
+/// The state after an auth response has been sent by a responder/received by
+/// an initiator, but no further communication can occur. A new handshake must
+/// be created in order to continue communication.
+pub struct Terminated;
```

Comment:
Definitely open to naming suggestions here. `Terminated` was one of the first things that came to mind.

Reply:
`Terminated` sounds OK, but I think it's helpful to think about why this would be the case.

Reply:
Can you elaborate on 
> I think it's helpful to think about why this would be the case.

I'm not quite following